From 0708992d1541e115375c23b11f06e29b4143f991 Mon Sep 17 00:00:00 2001
From: "Simental Magana, Marcos" <marcos.simental.magana@intel.com>
Date: Mon, 28 Sep 2015 17:25:56 -0500
Subject: [PATCH] default config

---
 etc/keystone.conf.sample | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/etc/keystone.conf.sample b/etc/keystone.conf.sample
index a9156dd..08ec2d4 100644
--- a/etc/keystone.conf.sample
+++ b/etc/keystone.conf.sample
@@ -326,7 +326,7 @@
 # (keystone.cache.memcache_pool) or Redis (dogpile.cache.redis) be used in
 # production deployments.  Small workloads (single process) like devstack can
 # use the dogpile.cache.memory backend. (string value)
-#backend = keystone.common.cache.noop
+backend = keystone.cache.memcache_pool
 
 # Arguments supplied to the backend module. Specify this option once per
 # argument to be passed to the dogpile.cache backend. Example format:
@@ -1905,7 +1905,7 @@
 # Entrypoint for the token persistence backend driver in the
 # keystone.token.persistence namespace. Supplied drivers are kvs, memcache,
 # memcache_pool, and sql. (string value)
-#driver = sql
+driver = memcache
 
 # Toggle for token system caching. This has no effect unless global caching is
 # enabled. (boolean value)
-- 
2.4.3

