From f9a935c4bf4afd8406ca8734c8a3b5cfdb6da020 Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Thu, 31 Mar 2016 16:02:44 -0600
Subject: [PATCH] Remove admin_token_auth

---
 etc/keystone-paste.ini | 11 +++--------
 1 file changed, 3 insertions(+), 8 deletions(-)

diff --git a/etc/keystone-paste.ini b/etc/keystone-paste.ini
index 4f3b0a2..0e2ee36 100644
--- a/etc/keystone-paste.ini
+++ b/etc/keystone-paste.ini
@@ -12,11 +12,6 @@ use = egg:keystone#build_auth_context
 [filter:token_auth]
 use = egg:keystone#token_auth
 
-[filter:admin_token_auth]
-# This is deprecated in the M release and will be removed in the O release.
-# Use `keystone-manage bootstrap` and remove this from the pipelines below.
-use = egg:keystone#admin_token_auth
-
 [filter:json_body]
 use = egg:keystone#json_body
 
@@ -51,17 +46,17 @@ use = egg:keystone#admin_service
 [pipeline:public_api]
 # The last item in this pipeline must be public_service or an equivalent
 # application. It cannot be a filter.
-pipeline = cors sizelimit url_normalize request_id admin_token_auth build_auth_context token_auth json_body ec2_extension public_service
+pipeline = cors sizelimit url_normalize request_id build_auth_context token_auth json_body ec2_extension public_service
 
 [pipeline:admin_api]
 # The last item in this pipeline must be admin_service or an equivalent
 # application. It cannot be a filter.
-pipeline = cors sizelimit url_normalize request_id admin_token_auth build_auth_context token_auth json_body ec2_extension s3_extension admin_service
+pipeline = cors sizelimit url_normalize request_id build_auth_context token_auth json_body ec2_extension s3_extension admin_service
 
 [pipeline:api_v3]
 # The last item in this pipeline must be service_v3 or an equivalent
 # application. It cannot be a filter.
-pipeline = cors sizelimit url_normalize request_id admin_token_auth build_auth_context token_auth json_body ec2_extension_v3 s3_extension service_v3
+pipeline = cors sizelimit url_normalize request_id build_auth_context token_auth json_body ec2_extension_v3 s3_extension service_v3
 
 [app:public_version_service]
 use = egg:keystone#public_version_service
-- 
2.5.5

