From 9c6c784ce682ae0d169550befe29d4da870f97a2 Mon Sep 17 00:00:00 2001
From: "Munoz, Obed N" <obed.n.munoz@intel.com>
Date: Wed, 18 Nov 2015 17:41:00 -0600
Subject: [PATCH] Set default syslog

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
---
 etc/keystone.conf.sample | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/keystone.conf.sample b/etc/keystone.conf.sample
index a9156dd..b6eba7e 100644
--- a/etc/keystone.conf.sample
+++ b/etc/keystone.conf.sample
@@ -134,7 +134,7 @@
 
 # Use syslog for logging. Existing syslog format is DEPRECATED and will be
 # changed later to honor RFC5424. (boolean value)
-#use_syslog = false
+use_syslog = true
 
 # (Optional) Enables or disables syslog rfc5424 format for logging. If enabled,
 # prefixes the MSG part of the syslog message with APP-NAME (RFC5424). The
-- 
2.6.2

