From cb1a01152de7c91d535c0808735f833151ca49ae Mon Sep 17 00:00:00 2001
From: Alberto Murillo <alberto.murillo.silva@intel.com>
Date: Mon, 14 Mar 2016 11:54:46 -0600
Subject: [PATCH] Set default syslog

Signed-off-by: Munoz, Obed N <obed.n.munoz@intel.com>
Signed-off-by: Alberto Murillo <alberto.murillo.silva@intel.com>
---
 etc/keystone.conf.sample | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/keystone.conf.sample b/etc/keystone.conf.sample
index ff62577..19ed0c4 100644
--- a/etc/keystone.conf.sample
+++ b/etc/keystone.conf.sample
@@ -157,7 +157,7 @@
 # Use syslog for logging. Existing syslog format is DEPRECATED and will be
 # changed later to honor RFC5424. This option is ignored if log_config_append
 # is set. (boolean value)
-#use_syslog = false
+use_syslog = true
 
 # Syslog facility to receive log lines. This option is ignored if
 # log_config_append is set. (string value)
-- 
2.5.0

